src/GovUk.Education.ExploreEducationStatistics.Notifier/local.settings.json

```diff
@@ -5,7 +5,8 @@
     "FUNCTIONS_WORKER_RUNTIME": "dotnet",
     "NotifyApiKey": "change-me",
     "VerificationEmailTemplateId": "change-me",
-    "PublicationNotificationEmailTemplateId": "change-me",
+    "OriginalReleasePublicationEmailTemplateId": "change-me",
+    "AmendedReleasePublicationEmailTemplateId": "change-me",
```

Comment:
These keys are a bit wordy.

Not saying we 100% need to change these, but it'd be a nice touch to shorten these down to something like:

- `ReleaseEmailTemplateId`
- `ReleaseAmendmentEmailTemplateId`

Including the word 'publication' is a bit redundant in the context of a release (a release by its definition always involves a publication). 

It's also a good idea to name things so that they sit close together when arranged alphabetically to make things easier to find. We can do this by keeping the less specific nouns first and adding progressively more specific nouns later (almost how you would arrange files in folders and similar to how we name components in the frontend).